js/app.js

```diff
@@ -0,0 +1,36 @@
+"use strict";
+
+(function(){
+  angular
+  .module("radioBip", [
+    "ui.router",
+    "songs"
+  ])
+
+  .config([
+    "$stateProvider",
+    RouterFunction
+  ])
+
+  function RouterFunction($stateProvider){
+    $stateProvider
+    .state("radioBipIndex", {
+      url: "/songs",
+      templateUrl: "js/index.html",
+      controller: "IndexController",
+      controllerAs: "SongsIndexViewModel"
+    })
+    .state("songsShow", {
```

Comment:
I would use consistent naming between states that involve songs (e.g., `songsIndex` + `songsShow`).